app/controllers/contents_controller.rb

```diff
@@ -0,0 +1,31 @@
+class ContentsController < ApplicationController
+  layout 'dashboard'
+  before_action :authenticate_user!
+
+  def add_content
+    @content = Content.new({})
+  end
+
+  def create_content
+    @lesson = Lesson.find_by(id: params[:lesson_id])
+    authorize @lesson, :create_lesson?, policy_class: LessonPolicy
+    @content = @lesson.contents.build(content_params)
+    if @content.content_type != "text" && params[:files].present?
```

Comment:
can we use constant for the string "text" here?

Reply:
I resolved it
Thank you